core/domain/acl_decorators_test.py

```diff
@@ -424,30 +425,39 @@ def setUp(self):
         rights_manager.publish_exploration(self.owner, self.published_exp_id)
 
     def test_guest_cannot_comment_on_feedback_threads(self):
-        response = self.mock_testapp.get(
-            '/mock/%s.thread1' % self.private_exp_id, expect_errors=True)
-        self.assertEqual(response.status_int, 302)
-        response = self.mock_testapp.get(
-            '/mock/%s.thread1' % self.published_exp_id, expect_errors=True)
-        self.assertEqual(response.status_int, 302)
+        with self.swap(constants, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', True):
```

Comment:
Do we need to test the False case, or do these handlers simply not exist in that case?

Reply:
Hmmm. I missed these earlier. They do exist in the false case too. Will add the tests.

Reply:
Done